File: 1-js-basics/3.2-conditional-statements-advanced-exercise/05-journey.js
function solve([budget, season]) {

    budget = Number(budget);

    let destination = '';

    let place = '';

    let total = '';

    if (budget <= 100) {

        destination = 'Bulgaria';

        season == 'summer' ? total = budget * 0.30 : total = budget * 0.70;

    } else if (budget > 100 && budget <= 1000) {

        destination = 'Balkans';

        season == 'summer' ? total = budget * 0.40 : total = budget * 0.80;

    } else {

        destination = 'Europe';

        total = budget * 0.90;

    }

    season == 'summer' && destination != 'Europe' ? place = 'Camp' : place = 'Hotel';

    return `Somewhere in ${destination}\n${place} - ${total.toFixed(2)}`;

}

console.log(solve(['50', 'summer']));
console.log(solve(['75', 'winter']));
console.log(solve(['312', 'summer']));
console.log(solve(['678.53', 'winter']));
console.log(solve(['1500', 'summer']));

File: 2-python/1-python-basics/3.2-conditional-statements-advanced-exercise/05-journey.py
budget = float(input())
season = input()
destination = ''
place_to_stay = ''
place_price_percent = 0

if budget <= 100:
    destination = 'Bulgaria'
    if season == 'summer':
        place_price_percent = 30
    else:
        place_price_percent = 70
elif budget <= 1000:
    destination = 'Balkans'
    if season == 'summer':
        place_price_percent = 40
    else:
        place_price_percent = 80
else:
    destination = 'Europe'
    place_price_percent = 90

if season == 'summer' and destination != 'Europe':
    place_to_stay = 'Camp'
else:
    place_to_stay = 'Hotel'

place_price = budget * place_price_percent / 100

print(f'Somewhere in {destination}\n{place_to_stay} - {place_price:.2f}')
